src/auth/index.ts

```diff
@@ -0,0 +1,83 @@
+import express, { Request } from 'express'
+import cors from 'cors'
+import * as yup from 'yup'
+import * as jwt from 'jsonwebtoken'
+
+import { table, client } from '../index'
+
+export const router = express.Router()
+
+router.use(cors(
+    {
+        credentials: true
+    }
+))
+router.use(express.json())
+
+router.get('/', (req, res) => {
+    res.json({
+        message: '🗝'
+    })
+})
+
+router.post('/login', async function (req, res, next) {
+    const isValidInput = await validateLogin(req.body).catch(err => {
+        res.status(401).json({
+            message: err.message
+        })
+
+    })
+
+    await table
+    const selectEmail = "select * from users where email = $1"
+    const emailParams = [req.body.email]
+
+    const queryResult =
+        await client.query(selectEmail, emailParams)
+
+    if (queryResult.rows.length <= 0) {
+        // send an error to the client
+        res.status(401).json({
+            message: 'Authorization Failed ⛔️'
+        })
+    }
+
+    if (queryResult.rows[0].password === req.body.password) {
```

Comment:
@Alexandria Now that I've looked at it a little closer I feel like we should be able to combine these two if statements on line `38` and `45`  together and have one else statement that sends back the error.